channels.go

```diff
@@ -39,14 +39,12 @@ type Channel struct {
 	Creator            string         `json:"creator"`
 	IsArchived         bool           `json:"is_archived"`
 	IsGeneral          bool           `json:"is_general"`
-	IsGroup            bool           `json:"is_group"`
-	IsStarred          bool           `json:"is_starred"`
```

Comment:
Yes; go and export all your channels and observe no IsStarred in the JSON, even with starred channels.